tedana/workflows/tedana.py

```diff
@@ -417,6 +457,16 @@ def tedana_workflow(
         The file must be a TSV file with the same number of rows as the number of volumes in
         the input data. Each column in the file will be treated as a separate regressor.
         Default is None.
+    ica_method : {'robustica', 'fastica'}, optional
```

Comment:
```suggestion
    ica_method : {'fastica', 'robustica'}, optional
```

Reply:
I think it makes more sense to have `fastica` listed first as it's the default option at the moment.